docs/quickstarts/nextjs.mdx

```diff
@@ -11,7 +11,7 @@ Learn how to use Clerk to quickly and easily add secure authentication and user
 
 ### Install `@clerk/nextjs`
 
-Once you have a Next.js application ready, you need to install Clerk's Next.js SDK. This gives you access to prebuilt components and hooks, as well as [helpers](/docs/references/nextjs/overview) for Next.js API routes, server-side rendering, and middleware.
+Once you have a Next.js application ready, you need to install Clerk's Next.js SDK. This gives you access to prebuilt components and hooks, as well as [helpers](/docs/references/nextjs/overview) for Next.js API routes, server-side rendering, and Middleware.

```

Comment:
Why capitalize middleware?

Reply:
Bryce pointed out we should to align with the Next docs, which capitalize.